crates/dbsp/src/circuit/runtime.rs

```diff
@@ -228,13 +263,29 @@ impl RuntimeInner {
             )));
         }
 
+        let cache_size_bytes = if let Some(storage) = &config.storage {
+            storage
+                .options
+                .cache_mib
+                .unwrap_or(nworkers * 256)
+                .saturating_mul(1024 * 1024)
```

Comment:
should this be done using usize?

Reply:
I think it is usize already (options.cache_mib is Option<usize>). What type do you think it should be?

Reply:
I didn't think Rust will allow i32 constants to be used in usize arithmetic. Maybe it does

Reply:
It infers the types of the constants.